scripts/setup-tools.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+echo "Set up tooling dependencies globally"
+
+PROTOC_VERSION=${PROTOC_VERSION:-27.0}
```

Comment:
NIT: I would put the default in the Makefile if possible, so all versioning is in one file.